Build failure: openscad

## Steps To Reproduce

Steps to reproduce the behavior:

1. build *openscad* on unstable

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
src/FileModule.cc: In member function 'void FileModule::registerUse(std::string, const Location&)':
src/FileModule.cc:68:25: error: 'is_regular' is not a member of 'fs'; did you mean 'is_regular_file'?
   68 |                 if (fs::is_regular(path)) {
      |                         ^~~~~~~~~~
      |                         is_regular_file
````

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

commit: c8365111999b8e0f268de233ce0e967062d6128c

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

## Notify maintainers
@bjornfor @gebner 
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/graphics/openscad/default.nix
+++ b/pkgs/applications/graphics/openscad/default.nix
@@ -64,6 +64,11 @@ mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace src/FileModule.cc \
+      --replace-fail 'fs::is_regular' 'fs::is_regular_file'
+  '';
+
   nativeBuildInputs = [
     bison
     flex

